### PR TITLE
fly console: keep stdout clean

### PR DESCRIPTION
**Pull request by benwaffle**
This allows piping the output to other commands

https://community.fly.io/t/suppress-log-output-when-running-fly-console-c/16535

### CODE REVIEW

File: internal/machine/ephemeral.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ func LaunchEphemeral(ctx context.Context, input *EphemeralInput) (*api.Machine,
 	if input.What != "" {
 		creationMsg += " " + input.What
 	}
-	fmt.Fprintf(io.Out, "%s.\n", creationMsg)
+	fmt.Fprintf(io.ErrOut, "%s.\n", creationMsg)
 
 	sp := spinner.Run(io, fmt.Sprintf("Waiting for %s to start ...", colorize.Bold(machine.ID)))
```
Comment (mootari) [inline internal/machine/ephemeral.go:45]: Doesn't the spinner text also end up in STDOUT?

Comment (rubys) [reply internal/machine/ephemeral.go:45]: Perhaps only output the creationMsg and spinner if the output is going to a terminal?

See: https://rderik.com/blog/identify-if-output-goes-to-the-terminal-or-is-being-redirected-in-golang/#final-thoughts

Comment (rubys) [reply internal/machine/ephemeral.go:45]: Now that I know what to look for, I found an existing helper function:

https://github.com/superfly/flyctl/blob/5dcfd7514a6b3b94bb9038453960bd33b5035fdc/helpers/stdin.go#L11-L17

Comment (benwaffle) [reply internal/machine/ephemeral.go:45]: weird that it doesn't use `isatty()`

Comment (benwaffle) [reply internal/machine/ephemeral.go:45]: https://github.com/superfly/flyctl/blob/master/internal/cmdutil/terminal.go#L9-L11

Comment (rubys) [reply internal/machine/ephemeral.go:45]: See https://github.com/superfly/flyctl/pull/3053 for a pull request that uses cmdutil.IsTerminal

**Pull request closed without merging**